docs/static/generated/rest_v1_dispatcher.yml

```diff
@@ -6,7 +6,7 @@ info:
   license:
     name: Apache 2.0
     url: https://www.apache.org/licenses/LICENSE-2.0.html
-  version: v1/1.18-SNAPSHOT
+  version: v1/1.19-SNAPSHOT
```

Comment:
This version was already updated in the master branch. Please rebase this pr to the latest master branch.